models/matsuo-lab/weblab-10b/harness.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+MODEL_NAME="weblab-10b"
+MODEL_ARGS="pretrained=matsuo-lab/${MODEL_NAME},torch_dtype=auto"
+TASK="jcommonsenseqa-1.1-0.3,jnli-1.1-0.3,marc_ja-1.1-0.3,jsquad-1.1-0.3,jaqket_v2-0.2-0.3,xlsum_ja-1.0-0.3,xwinograd_ja,mgsm-1.0-0.3"
```

Comment:
@kojima-takeshi188 For "base" models, `0.2` or `0.1` is fair to use. (If `0.3` is used for one base model, all base models in the leaderboard have to be evaluated with `0.3` for fair comparison and update the leaderboard.) 

Please refer to [this script](https://github.com/Stability-AI/lm-evaluation-harness/blob/jp-stable/models/stabilityai/stabilityai-japanese-stablelm-base-alpha-7b/harness.sh#L5).